pipeline-adapters/mlrun-pipelines-kfp-v1-8/src/mlrun_pipelines/mounts.py

```diff
@@ -144,52 +151,65 @@ def add_vol(name, mount_path, host_path):
         igz_version = mlconf.get_parsed_igz_version()
         if igz_version and igz_version >= semver.VersionInfo.parse("3.2.3-b1"):
```

Comment:
3.2.3 is dead this can be removed
```suggestion
        if igz_version:
```